rero_ils/modules/stats_cfg/jsonschemas/stats_cfg/stats_cfg-v0.0.1.json

```diff
@@ -0,0 +1,1061 @@
+                    }
+                  }
+                },
+                {
+                  "additionalProperties": false,
+                  "propertiesOrder": [
+                    "distributions",
+                    "filter"
+                  ],
+                  "required": [
+                    "type"
+                  ],
+                  "title": "Number of active patrons",
+                  "properties": {
+                    "type": {
+                      "title": "Type",
+                      "type": "string",
+                      "const": "number_of_active_patrons",
+                      "default": "number_of_active_patrons"
+                    },
+                    "distributions": {
+                      "allOf": [
+                        {
+                          "$ref": "#/definitions/distributions"
+                        },
+                        {
+                          "items": {
+                            "title": "distributions",
+                            "type": "string",
+                            "enum": [
+                              "time_range_month",
+                              "time_range_year",
+                              "library",
+                              "patron_postal_code",
+                              "gender"
+                            ]
+                          },
+                          "widget": {
+                            "formlyConfig": {
+                              "type": "selectWithSort",
+                              "options": [
+                                {
+                                  "value": "time_range_month",
+                                  "label": "time_range_month"
+                                },
+                                {
+                                  "value": "time_range_year",
+                                  "label": "time_range_year"
+                                },
+                                {
+                                  "value": "library",
+                                  "label": "library"
+                                },
+                                {
+                                  "value": "patron_postal_code",
+                                  "label": "patron_postal_code"
+                                },
+                                {
+                                  "value": "gender",
+                                  "label": "gender"
+                                }
+                              ]
+                            }
+                          }
+                        }
+                      ]
+                    },
+                    "filter": {
+                      "$ref": "#/definitions/filter"
+                    }
+                  }
+                }
+              ]
+            }
+          }
+        }
+      ]
+    }
+  },
+  "definitions": {
```

Comment:
very cosmetic : it seems convetions to place "definitions" before use it ; place `definitions` before main `properties`

Reply:
In the other schemes, the definition is always at the bottom of the file.